core/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 120,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": false,
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
Could we please enable this? It'll lead to smaller diffs in the future if devs decide to add an item at the end, for example (doesn't require adding a comma for the previous line)

Reply:
This is updated.